cli/npm/resolvers/local.rs

```diff
@@ -227,11 +227,13 @@ impl NpmPackageFsResolver for LocalNpmPackageResolver {
     permissions: &dyn NodePermissions,
     path: &Path,
   ) -> Result<(), AnyError> {
+    let mut cache = HashMap::new();
```

Comment:
I believe this will not cache anything - after this single call the cache will be discarded. The cache should probably live on the `LocalNpmPackageResolver`

Reply:
yea I am not sure where my head was at when doing this
will work on it later today